pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -175,6 +175,14 @@ def test_to_datetime_format_weeks(self):
 
 class TestToDatetime(object):
 
+    def test_to_datetime_iso_week_year_format(self):
```

Comment:
@jreback Can you please clarify this--> as I indicated above, need to exercise the paths in the code.?